src/docs/release/breaking-changes/bottom-navigation-title-to-label.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Bottom Navigation Title To Label
+description: Deprecated BottomNavigationBarItem’s title (a Widget) in favor of label (a String). 
+---
+
+## Summary
+
+`BottomNavigationBarItem.title` gives a deprecation warning, or no longer exists when referenced in 
+code.
+
+## Context
+
+`BottomNavigationBarItem`s `title` parameter was deprecated in favor of `label`. This change was 
+necessary to improve the user experience of `BottomNavigationBar`s when the text scale factor is 
+increased. Items in a `BottomNavigationBar` now show tooltips on long press. Accomplishing this 
+requires a `String` parameter on `BottomNavigationBarItem`s.
+
+
+## Description of change
+
+The `BottomNavigationBarItem` class has a `title` parameter, which is a `Widget`. This made it 
+impossible for the `BottomNavigationBar` to show `Tooltip` widgets, a change which was necessary to 
+improve the accessibility experience. Now, instead of taking `BottomNavigationBarItem.title` widget 
```

Comment:
=>
Now, instead of building the `BottomNavigationBarItem.title` widget, the `BottomNavigationBar` wraps the `BottomNavigationBarItem.label` in a `Text` widget and builds that.

Reply:
Done